tests/kernel/critical/testcase.yaml

```diff
@@ -4,6 +4,11 @@ common:
 tests:
   kernel.common:
     platform_exclude: nsim_sem_mpu_stack_guard nsim_em_mpu_stack_guard
+    filter: not CONFIG_WDT_SAM
+  kernel.common.sam:
+    platform_whitelist: arduino_due sam_e70_xplained sam4s_xplained
```

Comment:
Why did you switch to a whitelist for this case? Could you instead do:
```
filter: CONFIG_WDT_SAM
```

Reply:
I suppose, I can, @MaureenHelm , thanks for looking at it

Reply:
Done, pls, re-check, @MaureenHelm